### DISCUSSION TITLE
Appending text to the release body

### DISCUSSION THREAD

**Discussion by elect86** [ncipollo/release-action#64] [General]
Hi,

as titled, I'd like to append text to the release body, since I'm building nightly's, I'd like each platform to report the number of failing tests directly in the body

However the current implementation looks like is replacing it in toto

**Comment by ncipollo** [comment]
Appending text to the release body is a bit out of scope for the release action itself, but you should be able to achieve via a shell script action in the workflow. I think this easiest solution would be:
- Create a file in your repo which represents the nightly build body (something like `nightly.md`).
- As part of your workflow, append whatever output you want to `nightly.md`. For examples`run_my_tests.sh >> nightly.md`
- Check that file in - `git add nightly.md;git commit -m "nightly bump"`
- Push - `git push` (This step is important for the next run of the action)
- Run the release action. Pass `nightly.md` into `bodyFile`.

Alternatively, you could fetch the release, extract the body, append to it, then pass it back into the release action though I'm not sure how complicated that would get - https://docs.github.com/en/rest/reference/repos#get-a-release-by-tag-name

**Reply by elect86** [reply]
Nice idea, I'll give it a try, thanks!
